reference-architecture/osp-dns/ansible/bind-server.yml

```diff
@@ -0,0 +1,85 @@
+---
+# Create a bind master
+- hosts: bind
+
+  vars:
+
+  tasks:
+    - name: firewall daemon
+      action: package name=firewalld state=installed
+
+    - name: firewall python module
```

Comment:
Rather than entries using action: can you make it look like the following

https://github.com/openshift/openshift-ansible-contrib/blob/master/roles/prerequisites/tasks/main.yaml